partiql-tests-schema-proposal.md

```diff
@@ -23,45 +23,82 @@ The following is an abstraction to describe current and future tests we will hav
     name: <string>,
     statement: <string>,
     <other fields relevant for corresponding test_category>
-    assert: <struct>
+    assert: <struct | <list<struct>>>
```

Comment:
nit: `<struct> | <list<struct>>`?

Reply:
Updated in latest commit b6ea14c. Also updated in a couple other places for `env`.